test/runtests.jl

```diff
@@ -216,6 +216,46 @@ end
 
 rm(joinpath(Documenter_root, "build"); recursive = true)
 
+# Try generating documentation with generate()
+
+pkg_generate = if VERSION >= v"0.5-"
+    Pkg.add("PkgDev")
```

Comment:
Can this be conditional on something like `isdir(Pkg.dir("PkgDev")) || Pkg.add("PkgDev")`? Otherwise the tests take quite a while to run for me whether `PkgDev` is installed or not.